tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -142,7 +144,7 @@ public function addProductsToOrderWithNewInvoiceAndTheFollowingDetails(string $o
         $data = $table->getRowsHash();
         $productName = $data['name'];
         /** @var array $productsMap */
-        $productsMap = $this->getQueryBus()->handle(new SearchProducts($productName, 1));
+        $productsMap = $this->getQueryBus()->handle(new SearchProducts($productName, 1, $this->currencyIsoCode));
```

Comment:
This is a weird use, the class field is set by one method, but reused by another So if someone tries to use `addProductsToOrderWithNewInvoiceAndTheFollowingDetails` without using `orderDoesNotContainProduct` It will crash
I see three possibilities:
- the class field is set via a dedicated method that will be used in a `Given` state in the scenario, setting it for all methods
- remove the class field and add the parameter on both functions
- remove the parameter in both functions and use `Context::getContext()->currency->iso_code)` just like in CartFeatureContext.php

Reply:
done ! I chose the third option